mergify_engine/clients/github.py

```diff
@@ -387,6 +392,13 @@ async def delete(  # type: ignore[override]
             url, **self._prepare_request_kwargs(api_version, oauth_token, **kwargs)
         )
 
+    async def graphql_post(self, query: str) -> typing.Any:
+        response = (await self.post("/graphql", json={"query": query})).json()
+        data = response.get("data")
+        if not data:
+            raise GraphqlError(response)
+        return response
```

Comment:
We should put the string into the exception for now, as we don't known that the error structure looks like.

```suggestion
        response = await self.post("/graphql", json={"query": query})
        data = response.json()
        if "data" not in data:
            raise GraphqlError(response.text)
        return data
```